### ISSUE TITLE
(legacy) [version 2.x] Some flags have dissapeared

### ISSUE THREAD

**Issue by ebaynaud** [catamphetamine/react-phone-number-input#391]
Due to: https://github.com/lipis/flag-icon-css/issues/860

**Comment by HasanCanUgurlu** [comment]
👋🏼  I've upgraded the package to `version 3.0.0` but it seems like it's still trying to fetch the icons from lipis's library. I don't see any requests from `country-flag-icons`. It shows flags if I use `flagUrl={`https://purecatamphetamine.github.io/country-flag-icons...` prop.  How can I debug this?

**Comment by catamphetamine** [comment]
> I've upgraded the package to version 3.0.0 but it seems like it's still trying to fetch the icons from lipis's library.

It shouldn't.

**Comment by n8sabes** [comment]
Indeed, I am seeing this as well. Dev and Production builds are pulling flags from `purecatamphetamine.github.io`.

**Comment by AomineVlady** [comment]
При обновлении с версии 1.1.4 до последней столкнулся с проблемой "слетели стили". Селект дефолтный и прочее. Могу я как-то обновиться оставив старый вид?
